src/main/scala/com/yotpo/metorikku/output/writers/file/FileOutputWriter.scala

```diff
@@ -52,9 +60,11 @@ class FileOutputWriter(props: Map[String, Object], outputFile: Option[File]) ext
     }
 
```

Comment:
@Irenez753 you can move this whole section to a new function for building the path and so you could probably remove the `// scalastyle:off method.length`